packages/ubuntu_wsl_setup/lib/main.dart

```diff
@@ -7,11 +9,26 @@ import 'app.dart';
 void main(List<String> args) {
   final options = parseCommandLine(args, onPopulateOptions: (parser) {
     parser.addFlag('reconfigure');
+    parser.addOption(
+      'prefill',
+      valueHelp: 'prefill',
+      help: '''
+Path of the YAML file containing prefill information, which
+feeds the installer with partial information to prefill the
+screens, yet allowing user to overwrite any of those during setup.
+  ''',
+    );
   })!;
   var variant = Variant.WSL_SETUP;
+  List<String>? serverArgs;
   if (options['reconfigure'] == true) {
     variant = Variant.WSL_CONFIGURATION;
   }
+  if (options['prefill'] != null) {
+    final prefillFile = options['prefill'];
+    final fileExists = File(prefillFile).existsSync();
+    if (fileExists) serverArgs = ['--prefill', options['prefill']];
```

Comment:
should we print an error if the file doesn't exist, to avoid silently ignoring the option?

how does the server handle it if the file doesn't exist? I'm wondering if simply passing the option through and letting the server throw an error would result in a clear enough error message?